Source/Workers/createVerticesFromQuantizedTerrainMesh.js

```diff
@@ -94,6 +104,18 @@ define([
         var indexBuffer = IndexDatatype.createTypedArray(quantizedVertexCount + edgeVertexCount, indexBufferLength);
         indexBuffer.set(parameters.indices, 0);
 
+        var occludeePointInScaledSpace;
+        var orientedBoundingBox;
+        var boundingSphere;
+
+        if (exaggeration > 1.0) {
```

Comment:
>Why not !== 0.0? Could the bounding volume be smaller in that case?

`1.0` is the default (the volumes that are already computed for the tile). Changing it to `!== 1.0` also fixed the issue blow when setting it to `0.1`